plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveConfig.java

```diff
@@ -502,6 +503,19 @@ public HiveConfig setMaxPartitionBatchSize(int maxPartitionBatchSize)
         return this;
     }
 
+    public Optional<String> getGlueProxyApiId()
+    {
+        return glueProxyApiId;
+    }
+
+    @Config("hive.metastore.glue.proxy-api-id")
```

Comment:
Why not `GlueHiveMetastoreConfig`? Also, could you update "AWS Glue catalog configuration properties" in hive.rst? 

(I know it comes internal code though)

Reply:
Totally missed that. Thanks.